packages/validator-ajv8/src/validator.ts

```diff
@@ -234,17 +235,19 @@ export default class AJV8Validator<
         ).title;
 
         if (uiSchemaTitle) {
-          message = message?.replace(currentProperty, uiSchemaTitle);
+          message = message.replace(currentProperty, uiSchemaTitle);
         } else {
-          const parentSchemaTitle =
-            parentSchema?.[PROPERTIES_KEY]?.[currentProperty]?.title;
+          const parentSchemaTitle = get(
+            parentSchema,
+            `${PROPERTIES_KEY}.${currentProperty}.title`
+          );
```

Comment:
You could also use the array notation for the props instead saving the need to construct a string, i.e.:
```
const parentSchemaTitle = get(
            parentSchema,
            [PROPERTIES_KEY, currentProperty, "title"]
          );
```